demo-app/src/main/java/org/openmined/syft/demo/StandaloneDemo.kt

```diff
@@ -9,7 +10,13 @@ import org.openmined.syft.threading.ProcessSchedulers
 
 @ExperimentalUnsignedTypes
 fun main() {
-    val schedulers = object : ProcessSchedulers {
+    val networkingSchedulers = object : ProcessSchedulers {
```

Comment:
Do we need two different sets of schedulers? If using just one, wouldn't it work as expected?

Reply:
The issue is we need different type of thread scheduling for computation and networking. We can spawn multiple threads for networking however compute usually would be restricted to single thread.